app/presenters/v2/buildpack_presenter.rb

```diff
@@ -0,0 +1,17 @@
+module CloudController
+  module Presenters
+    module V2
+      class BuildpackPresenter < DefaultPresenter
+        extend PresenterProvider
+
+        present_for_class 'VCAP::CloudController::Buildpack'
+
+        def entity_hash(controller, buildpack, opts, depth, parents, orphans=nil)
+          entity = super
+          entity['filename'] = "#{buildpack.filename} (#{buildpack.stack})".strip
```

Comment:
Why are we setting filename to include the stack? This feels like something that should happen in the CLI instead of the API.

Reply:
This was primarily to make things clearer to folks who might be using an older CLI which isn't aware of the multiple stack support -- otherwise they'll just see what appears to be duplicate buildpacks. If there's a better way we can handle this, we'd love to discuss.